events/friends_of_kenilworth_gardens.py

```diff
@@ -1,14 +1,18 @@
-import requests
-import os
 import datetime
+import logging
+import os
+import requests
+
+logger = logging.getLogger(__name__)
 
 FONA_EVENTBRITE_ORG_ID = 8632128868
 # For a local run, be sure to create an env variable with your Eventbrite token
 # For example:
 # $ export EVENTBRITE_TOKEN=<EVENTBRITE TOKEN Key>
 try:
     EVENTBRITE_TOKEN = os.environ['EVENTBRITE_TOKEN']
-except KeyError:
+except Exception as e:
```

Comment:
We should keep the `KeyError` catch, as that's the only error that'll raise if the environment variable isn't there and we need the user to input it. Adding `except Exception as e` after the `KeyError` check would be good, but then that would be a `logger.critical` instead of `logger.error` as we wouldn't be able to do anything else in this script without the token. It's also good to add the `exc_info = True` kwarg to the `logger.critical()` call as that'll log the stack trace. You can also string interpolate the `e` into the message, e.g. `logger.critical(f"Error {e} getting API token", exc_info = True)`.